Sources/Core/Client.swift

```diff
@@ -68,14 +68,24 @@ public enum SyncMode {
     case pushOnly
 }
 
-struct Attachment {
+class Attachment {
```

Comment:
`struct` seems to work fine. Is there any reason for the change?

Reply:
The Attachment is used as a parameter of syncInternal(). In syncInternal()  check the value of attachment.realSyncMode which can be changed by another API call. If we use struct, the copied attachment is passed to syncInternal() So the change of the value of attachment.realSyncMode is not affected.